### DISCUSSION TITLE
How to enable manual linking locally?

### DISCUSSION THREAD

**Discussion by tkrunning** [supabase/supabase#22214] [Questions]
How can I enable manual linking (with `supabase.auth.linkIdentity()`) for my local dev environment? When calling the function I get a 404 with this response:

```
{
    "code": 404,
    "msg": "Manual linking is disabled"
}
```

I've tried adding this to my `.env`: `GOTRUE_SECURITY_MANUAL_LINKING_ENABLED="true"` but it didn't do anything.

Could it be that my GoTrue version is too old? When I run `supabase start` it says that my local service versions don't match the linked project:

```
You are running outdated service versions locally:
postgrest/postgrest:v12.0.1 => v10.2.0
supabase/storage-api:v0.46.4 => v0.48.2
supabase/postgres:14.1.0.89 => 14.1.0.69
supabase/gotrue:v2.132.3 => v2.144.0
Run supabase link to update them.
```

However, running `supabase link` does not update the versions... How can I update the versions?

**Comment by tkrunning** [comment]
I figured it out, it's possible to set in `config.toml`. The only place it seems to be documented at the moment is here: https://github.com/supabase/supabase/blob/8f950a111e1bf24e89d0010435ec71e9ff6dcfed/playwright-tests/supabase/config.toml#L85

For anyone else looking for this, the config you'll need to set is `enable_manual_linking = true` (added under `auth`).

**Reply by vikito755** [reply]
Thanks a ton!
